src/applications/vaos/containers/VAFacilityPage.jsx

```diff
@@ -48,18 +48,6 @@ const uiSchema = {
     'ui:title':
       'Appointments are available at the following locations. Some types of care are only available at certain locations. Please choose your preferred location.',
     'ui:widget': 'radio',
-    'ui:validations': [
```

Comment:
I don't think this is ever triggered, and should be covered by the enum list default validation